src/DIRAC/Resources/Storage/StorageElement.py

```diff
@@ -157,19 +157,19 @@ class StorageElementItem(object):
         "getDirectory": {"localPath": False},
     }
 
-    def __init__(self, name, plugins=None, vo=None, hideExceptions=False):
+    def __init__(self, name, protocolSections=None, vo=None, hideExceptions=False):
```

Comment:
This is a rather important change that should be very well advertised.

Reply:
yes for sure. If it gets merged, I'll add it to the twiki